client/yarn.lock

```diff
@@ -9,18 +9,18 @@
   dependencies:
     "@babel/highlight" "^7.0.0"
 
-"@babel/core@^7.1.0":
```

Comment:
Looks like there were some merge conflicts that resulted in some packages being incorrectly downgraded. Can you delete this file locally and run `yarn install` in the client folder to generate a new file?